### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Volodymyr-Mykychak**

### CODE REVIEW

File: src/main/java/com/store/book/model/Book.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "books")
+@Data
+public class Book {
```
Comment (Rommelua) [inline src/main/java/com/store/book/model/Book.java:11]: add constraints
image

File: src/main/java/com/store/book/model/CartItem.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.store.book.model;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Entity
+@Data
+public class CartItem {
```
Comment (Rommelua) [inline src/main/java/com/store/book/model/CartItem.java:8]: All models except Book are out of task scope - delete them

File: target/classes/application.properties
Comment (olekskov) [inline target/classes/application.properties:1]: Target folder should be excluded from git.
Please remove and recheck .gitignore file

File: README.md [hunk 3]
```diff
@@ -1 +1,158 @@
-# bookstore
\ No newline at end of file
+We are going to implement an app for Online Book store. We will implement it step by step. In this app we will have the
```
Comment (olekskov) [inline README.md:1]: can't see any obvious reason to put full task description in readme

**Pull request merged**